docs/build/windows.md

```diff
@@ -1,3 +1,328 @@
+   prefer.
+
+### Web Dependencies
+
+**Projects**:
+* Keyman Developer
+* Keyman for Android
+* KeymanWeb
+
+**Requirements**:
+* node.js 14+
+* emscripten 2.0.23+
+* wasm-pack 0.9.1+
+* openjdk 8+
+
+```ps1
+# Elevated PowerShell
+rustup target add i686-pc-windows-msvc
+rustup target add wasm32-unknown-unknown
+choco install nodejs emscripten
+choco install openjdk
+
+# for *much* faster download, hide progress bar (PowerShell/PowerShell#2138)
+$ProgressPreference = 'SilentlyContinue'
+Invoke-WebRequest -Uri https://github.com/rustwasm/wasm-pack/releases/download/v0.9.1/wasm-pack-init.exe -OutFile $env:TEMP\wasm-pack-init.exe
+& "$env:TEMP\wasm-pack-init.exe"
+```
+
+**Environment variables**:
+* `PATH`:
+  * Add emscripten to path, most likely %LocalAppData%\emsdk\upstream\emscripten
+
+### Windows Platform Dependencies
+
+**Projects**:
+* Keyman Developer
+* Keyman for Windows
+
+**Requirements**:
+* Delphi 10.3 Community or Professional:
+  https://www.embarcadero.com/products/delphi/starter/free-download (Delphi
+  Windows Community, DUnit Unit Testing Frameworks required)
+
+  * Note: Delphi 10.3 Community is no longer available. Delphi 10.4 Community no
+    longer includes command line compilers. This change means that building
+    Keyman with Delphi 10.4 Community is not really viable. This means you can
+    only really use the Professional Edition, which can be used on a trial basis
+    for a short time. (We are actively working to remove Delphi dependencies
+    given the licensing issues with using it.)
+
+  Start Delphi IDE once after installation as it will create various environment
+  files and take you through required registration.
+
+* Visual C++ 2019 Community or Professional
+
+  ```ps1
+  choco install visualstudio2019community visualstudio2019-workload-nativedesktop visualstudio2019buildtools
+  ```
+
+  Recommended: configure Visual Studio to use two-space tab stops:
+  1. Open the options dialog: Tools > Options.
+  2. Navigate to Text Editor > All Languages > Tabs.
+  3. Change 'Tab size' to 2 and 'Indent size' to 2.
+  4. Select 'Insert spaces'.
+
+**Required environment variables**:
+* `PATH`
+  * Add the C:\Projects\keyman\keyman\windows\lib folder in the Keyman
+    repository to your `PATH` environment variable. This is required for
+    Keyman's design-time packages to load in Delphi.
+* [`KEYMAN_CEF4DELPHI_ROOT`](#keyman_cef4delphi_root)
+
+**Optional environment variables**:
+* [`GIT_BASH_FOR_KEYMAN`](#git_bash_for_keyman)
+* [`USERDEFINES`](#userdefines)
+
+```bat
+SET KEYMAN_CEF4DELPHI_ROOT=c:\Projects\keyman\CEF4Delphi_Binary
+SET GIT_BASH_FOR_KEYMAN="C:\Program Files\Git\bin\bash.exe" --init-file "c:\Program Files\Git\etc\profile" -l
+```
+
+**Additional requirements for release builds**:
+* [7-Zip](http://www.7-zip.org/), used for archiving build files
+* [HTML Help Workshop](https://www.microsoft.com/en-us/download/details.aspx?id=21138)
```

Comment:
Regrettably, Microsoft's removed that link.

Reply:
```suggestion
* [HTML Help Workshop](http://web.archive.org/web/20160201063255/http://download.microsoft.com/download/0/A/9/0A939EF6-E31C-430F-A3DF-DFAE7960D564/htmlhelp.exe) note: Microsoft no longer offer this download...
```